riak.cabal

```diff
@@ -122,6 +123,7 @@ library
                 random-shuffle                >= 0.0.4 && < 0.1,
                 riak-protobuf                 == 0.21.*,
                 semigroups                    >= 0.16,
+                stm,
                 text                          == 1.2.*,
```

Comment:
Could I get you to put some version ranges on `stm` and `transfromers-base` please?